eox_tenant/models.py

```diff
@@ -177,3 +180,27 @@ class Meta:
         Model meta class.
         """
         app_label = "eox_tenant"
+
+class EdnxCourseEnrollmentManager(CourseEnrollmentManager):
+    """
+    Custom manager for CourseEnrollment with Table-level filter methods.
+    """
+    def get_queryset(self):
+
+        orgs_to_include = get_microsite().get_value('course_org_filter')
+
+        # Make sure we have a list
+        if orgs_to_include and not isinstance(orgs_to_include, list):
+            orgs_to_include = [orgs_to_include]
+
+        if not orgs_to_include:
+            # Making orgs_to_include an empty iterable
+            orgs_to_include = []
```

Comment:
I think that in this case we have to use something like:
`orgs_to_exclude = configuration_helpers.get_all_orgs()
   return super(CourseEnrollmentManager, self).get_queryset().exclude(course_id__org__in=orgs_to_exclude)

`